guides/jupyter-notebooks.md

```diff
@@ -1,47 +1,114 @@
-# Jupyter Notebooks
+# Jupyter Notebooks and Tutorials
 
-Jupyter Notebooks are a convenient format for creating and sharing documents that combine code, data analyses, visualizations, and prose. This guide describes best practices for creating readable (easy to understand), portable (likely to work on many computers) notebooks. Following this guide is a requirement for those authors contributing content to the [STScI notebooks repository](https://github.com/spacetelescope/notebooks).
+This guide describes best practices for creating readable (easy to understand),
+portable (likely to work on many computers) tutorials. Jupyter Notebooks are a
```

Comment:
Sorry this is partly backing off what @barentsen said earlier, but I'm not sure "tutorial" is quite the right general term here.  The reason why is that this style guide *also* applies to some notebooks that are less "pedagogical" and more like "this notebook yields a specific science result". So I think it's better to lead off with the original wording about "a convenient format..." and then add at the end of this paragraph something like "this style guide, when not documenting notebook-specific syntax, also applies to other formats like non-notebook tutorial or example descriptions." ?